.github/workflows/ci.yml

```diff
@@ -29,9 +29,11 @@ jobs:
       - uses: actions/checkout@v3
       - uses: actions/setup-node@v3
         with:
-          node-version: 18
+          node-version: 18.15.0
```

Comment:
If you use only the major version number, the latest in the major version is selected